### ISSUE TITLE
Require register exe version?

### ISSUE THREAD

**Issue by Deltafox79** [DoubleLabyrinth/Xmanager-keygen#1]
Hi,
But to use the generated serial, do you need the complete installation file and not the downloadable evaluation version? Where do you find the latest versions?

Greetings

**Comment by DoubleLabyrinth** [comment]
I download Xmanager from its official website https://www.netsarang.com/download/main.html

I think this software is just like Navicat. Though the version that the official website offers is evaluation version, you can use all of its features without any limits except time, no features cut off. In other words, I think the downloadable evaluation version exactly is the complete installation version.

If there are any difference between complete installation and downloadable evaluation version, please tell me.

**Comment by Deltafox79** [comment]
There is a difference between the evalutation and the retail version. The evaluation version does not allow the insertion of the serial during the installation .. The retail version instead .. here I found the link for the latest retail version ..

https://cdn.netsarang.net/f6d42f07/XmanagerEnterprise-5.0.1249r.exe

Greetings

Greetings

https://cdn.netsarang.net/f6d42f07/XmanagerEnterprise-5.0.1249r.exe

Regards

**Comment by Deltafox79** [comment]
https://i.imgur.com/X41WPBU.jpg

Xmanager_Enterprise_v5.0.1249r_Incl_Keygen_v1.0 :

http://www119.zippyshare.com/v/GpTfpF11/file.html

Regards

**Comment by DoubleLabyrinth** [comment]
Thanks, I've checked retail version and evaluation version. And I found they have the same `nslicense.dll`. Because the entry of product key validation is `NSLICENSE_IsValidLicense` in `nslicense.dll`. So I think same `nslicense.dll`, same registration algorithm.

**Comment by Deltafox79** [comment]
Yes But the evalutation version does not allow the insertion of the serial and the term evalutation remains. Probably entering it manually in the Windows registry file should work ProductKey = xxxx

**Comment by DoubleLabyrinth** [comment]
Actually, you can input product key at `Xshell`/`Xftp` --> Help (in menu bar) --> Change Product Key

**Comment by alex6988** [comment]
hello, that link to download (http://www119.zippyshare.com/v/GpTfpF11/file.html) there is nothing, I have installed all the plugin but in the end nothing is downloaded, you could upload it to a google drive or mega?
